projects/js-packages/components/index.ts

```diff
@@ -14,74 +14,74 @@ along with this program; if not, write to the Free Software
-export { default as PricingCard } from './components/pricing-card';
-export { default as NumberSlider } from './components/number-slider';
-export { default as AdminSection } from './components/admin-section/basic';
-export { default as AdminSectionHero } from './components/admin-section/hero';
-export { default as AdminPage } from './components/admin-page';
-export { default as DecorativeCard } from './components/decorative-card';
-export { default as Col } from './components/layout/col';
-export { default as Testimonials } from './components/testimonials';
-export { default as Container } from './components/layout/container';
-export { default as useBreakpointMatch } from './components/layout/use-breakpoint-match';
-export { default as CopyToClipboard } from './components/copy-to-clipboard';
-export * from './components/icons';
-export { default as SplitButton } from './components/split-button';
-export { default as ThemeProvider } from './components/theme-provider';
-export { default as ThreatFixerButton } from './components/threat-fixer-button';
-export { default as ThreatSeverityBadge } from './components/threat-severity-badge';
-export { default as ThreatsDataViews } from './components/threats-data-views';
-export { default as Text, H2, H3, Title } from './components/text';
-export { default as ToggleControl } from './components/toggle-control';
-export { default as numberFormat } from './components/number-format';
-export { default as QRCode } from './components/qr-code';
-export { default as Button } from './components/button';
-export type { ButtonProps } from './components/button/types';
-export { default as LoadingPlaceholder } from './components/loading-placeholder';
-export { default as TermsOfService } from './components/terms-of-service';
-export { default as Chip } from './components/chip';
+export { default as JetpackLogo } from './components/jetpack-logo/index.js';
+export { default as JetpackSearchLogo } from './components/jetpack-search-logo/index.js';
+export { default as JetpackProtectLogo } from './components/jetpack-protect-logo/index.js';
+export { default as JetpackVaultPressBackupLogo } from './components/jetpack-vaultpress-backup-logo/index.js';
+export { default as JetpackVideoPressLogo } from './components/jetpack-videopress-logo/index.js';
+export { default as getRedirectUrl } from './tools/jp-redirect/index.js';
+export { default as getProductCheckoutUrl } from './tools/get-product-checkout-url/index.js';
+export { isFirstMonthTrial } from './tools/pricing-utils/index.js';
+export { default as AutomatticBylineLogo } from './components/automattic-byline-logo/index.js';
+export { default as AutomatticIconLogo } from './components/automattic-icon-logo/index.js';
+export { default as AutomatticForAgenciesLogo } from './components/automattic-for-agencies-logo/index.js';
+export { default as JetpackFooter } from './components/jetpack-footer/index.js';
+export { default as Spinner } from './components/spinner/index.jsx';
```

Comment:
If we get around to adding building, the file will wind up at `index.js`, not `index.jsx`.
```suggestion
export { default as Spinner } from './components/spinner/index.js';
```

Reply:
Done :+1: [04da209](https://github.com/Automattic/jetpack/pull/41524/commits/04da2096530298865d43ea4273357f7216bf2622)